index.html

```diff
@@ -3,10 +3,24 @@
   <head>
     <title>2048</title>
     <link rel="stylesheet" media="all" href="stylesheets/2048.css"/>
+    <link rel="stylesheet" media="all" href="stylesheets/animation.css"/>
```

Comment:
Nit, in case it's interesting: these trailing `/`s for self-closing tags were necessary for XHTML. This file claims to be HTML5, so you don't need it.